<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminSettingMenu</class>
 <widget class="QDialog" name="AdminSettingMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>30</y>
     <width>191</width>
     <height>201</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="RestaurantButton">
      <property name="text">
       <string>Restaurant Add/Delete/Reinitialize</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="Restaurant_Layout">
      <item>
       <widget class="QRadioButton" name="RestaurantAdd">
        <property name="text">
         <string>Add Restaurant</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="RestaurantDelete">
        <property name="text">
         <string>Delete Restaurant</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="Reinitialize">
        <property name="text">
         <string>Reinitialize First 10</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="ItemButton">
      <property name="text">
       <string>Item Add/Delete/Edit</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="Item_layout">
      <item>
       <widget class="QRadioButton" name="ItemAdd">
        <property name="text">
         <string>Add Item</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="ItemDelete">
        <property name="text">
         <string>Delete Item</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="ItemEdit">
        <property name="text">
         <string>Edit Item</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AdminSettingMenu</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AdminSettingMenu</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
